docs/samples/tensorflow/README.md

```diff
@@ -3,7 +3,7 @@
 ## Setup
 1. Your ~/.kube/config should point to a cluster with [KFServing installed](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#deploy-kfserving).
 2. Your cluster's Istio Ingress gateway must be network accessible.
-3. Your cluster's Istio Egresss gateway must [allow Google Cloud Storage](https://knative.dev/docs/serving/outbound-network-access/)
+3. Your cluster's Istio Egresss gateway must [allow Google Cloud Storage](https://knative.dev/v0.8-docs/serving/outbound-network-access/)
```

Comment:
is there a latest pointer? I am afraid this gets outdated quickly

Reply:
Jane, I think other page may be also has link error, please have a check and fix together. And agree with @yuzisun, any master/stable version, if not, we need take latest version as we are going to upgrading knative to 0.10.1 now.. thanks.

Reply:
@yuzisun Thanks for review! When I search for correct link, I notice "outbound-network-access" section exist only in v0.8, but NOT in v0.9-v.11(current/latest)  anymore. This is the reason I change the broken link to the "working" url. 

@jinchihe  will fix the other pages, if needed.

Reply:
Found "outbound-network-access" section is removed in knative doc starting v0.9 because of following PR: https://github.com/knative/docs/issues/1859

"Outbound access is allowed by default
The model changed in Istio 1.1 and OSS recommends Istio 1.2"
"enabled by default since Istio 1.1. Since we require 1.1 for Knative v0.8.1 onward, Knative > v0.8.1 would suffice"

As from https://www.kubeflow.org/docs/components/serving/kfserving/, KFserving pre-req
Knative Serving (v0.8.0 +) and Istio (v1.1.7+) , I believe we can remove the following line under "Setup":
3. Your cluster's Istio Egresss gateway must allow Google Cloud Storage

@yuzisun and @jinchihe What do you think?

Reply:
Istio 1.1 by default allows outbound access, I think we can just remove this too.

Reply:
Thanks Dan. Will remove.